src/pages/en/reference/api-reference.md

```diff
@@ -158,6 +158,29 @@ You could pass a callback function that renders our the message:
 <div>Hello world!</div>
 ```
 
+### `Astro.self`
+
+`Astro.self` allows Astro components to be recursively called. This behaviour lets you call the same Astro component over again by defining `<Astro.self>` within the component template.  Allowing you to easily iterate over large data stores and nested data-structures.
```

Comment:
```suggestion
`Astro.self` allows Astro components to be recursively called. This behaviour lets you call the same Astro component over again by defining `<Astro.self>` within the component template. 
```
Do you think maybe these two sentences say pretty much the same thing? Can this be condensed into one sentence?

b) And, I don't think I'd say we've "defined" Astro.self? I know you use "invoke" somewhere else. Depending on the sentence, maybe "called" or "rendered"? Something else?